### DISCUSSION TITLE
New mobile navbar

### DISCUSSION THREAD

**Discussion by blox814123** [Xyphyn/photon#309] [General]
The new navbar for desktop should be reverted to the old version. The new one is basically created for mobile navigation, and goes against all design principles for desktop navigation. It takes a lot of benefits of photon away.

**Comment by TailyFair** [comment]
I've used it now for quite a bit and can fully agree now. Muscle memory is too strong, at least home button should be moved where it exists in most of other systems 😄
Each day I'm confused why home button shows my profile

**Comment by prezmix** [comment]
Agreed with the above, how about an environment variable to toggle the old navbar design with the prominent (search, explore, + create, etc) buttons, site name/icon logo?
